### PR TITLE
Introduction of the Sex SVO as replacement for Gender

### PR DESCRIPTION
**Pull request by Corniel**
Due to name changes done by ISO, Qowaiv should follow that naming convention.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -207,6 +204,19 @@ var argentina = PostalCode.Parse("Z1230ABC");
 argentina.ToString("AR"); // Z 1230 ABC
 ```
 
+### Sex
+Represents a sex based on an ISO 5218 code.
+
+``` C#
+var sex = Sex.Female;
+var parsed = Sex.Parse("Male");
```
Comment (LauraXiulan) [inline README.md:212]: You don't do anything with this, should it have an example or commented out value?

File: specs/Qowaiv.Specs/Cast_specs.cs [hunk 2]
```diff
@@ -26,12 +26,6 @@ public class Invalid
         [TestCase(1_000_000_000)]
         public void int_to_HouseNumber(int num) => QAssert.InvalidCast(() => (HouseNumber)num);
 
-        [TestCase(int.MaxValue)]
-        [TestCase(1_000_000)]
-        [TestCase(-1)]
-        [TestCase(11)]
-        public void int_to_Gender(int num) => QAssert.InvalidCast(() => (Gender)num);
```
Comment (LauraXiulan) [inline specs/Qowaiv.Specs/Cast_specs.cs:33]: No int_to_Sex test?

File: specs/Qowaiv.Specs/Debug_SVO_specs.cs [hunk 3]
```diff
@@ -101,7 +101,6 @@ public class Displays
         [TestCase(typeof(Elo), "1600", 1600d)]
         [TestCase(typeof(EmailAddress), "svo@qowaiv.org", "svo@qowaiv.org")]
         [TestCase(typeof(Fraction), "-69/17", "-⁶⁹⁄₁₇ = -4.05882353")]
-        [TestCase(typeof(Gender), "F", "Female")]
```
Comment (LauraXiulan) [inline specs/Qowaiv.Specs/Debug_SVO_specs.cs:104]: No testCase for Sex?

File: specs/Qowaiv.Specs/Sex_specs.cs [hunk 4]
```diff
@@ -0,0 +1,562 @@
...
+        public void IsEmpty_returns(bool result, Sex svo)
+            => svo.IsEmpty().Should().Be(result);
+
+        [TestCase(false, "Male")]
+        [TestCase(false, "Female")]
+        [TestCase(true, "?")]
+        [TestCase(true, "")]
+        public void IsEmptyOrUnknown_returns(bool result, Sex svo)
+        {
+            Assert.AreEqual(result, svo.IsEmptyOrUnknown());
```
Comment (LauraXiulan) [inline specs/Qowaiv.Specs/Sex_specs.cs:33]: Inconsistent use of Asserts and Should

File: specs/Qowaiv.Specs/Sex_specs.cs [hunk 5]
```diff
@@ -0,0 +1,562 @@
...
+    }
+
+    public class Has_custom_formatting
+    {
+        [Test]
+        public void _default()
+        {
+            using (TestCultures.En_GB.Scoped())
+            {
+                Assert.AreEqual("Female", Svo.Sex.ToString());
```
Comment (LauraXiulan) [inline specs/Qowaiv.Specs/Sex_specs.cs:243]: Here as well, inconsistent use of Assert and Should

File: specs/Qowaiv.Specs/Sex_specs.cs [hunk 6]
```diff
@@ -0,0 +1,562 @@
...
+        [TestCase("Female", 2L)]
+        [TestCase("Female", 2d)]
+        public void convention_based_deserialization(Sex expected, object json)
+        {
+            var actual = JsonTester.Read<Sex>(json);
+            Assert.AreEqual(expected, actual);
+        }
+
+        [TestCase(null, "")]
+
```
Comment (LauraXiulan) [inline specs/Qowaiv.Specs/Sex_specs.cs:445]: Extra enter

File: specs/Qowaiv.Specs/Cast_specs.cs (hunk 2 above)
Comment (Corniel) [reply specs/Qowaiv.Specs/Cast_specs.cs:33]: Nope, handled by Sex_specs.

File: specs/Qowaiv.Specs/Sex_specs.cs [hunk 7]
```diff
@@ -0,0 +1,562 @@
...
+        [Test]
+        public void via_TypeConverter_registered_with_attribute()
+            => typeof(Sex).Should().HaveTypeConverterDefined();
+
+        [Test]
+        public void from_null_string()
+        {
+            using (TestCultures.En_GB.Scoped())
+            {
+                //Converting.From(null).To<Sex>().Should().Be(Sex.Empty);
```
[the remark this answers is not in the document]
Comment (Corniel) [reply specs/Qowaiv.Specs/Sex_specs.cs:393]: Can only be enabled after another pr. :S

**Pull request merged**